templates/service-template.yml

```diff
@@ -172,6 +172,11 @@ parameters:
   description: Enable the Kafka TLS certificate
   value: "false"
 
+- name: ENABLE_KAFKA_CNAME_REGISTRATION
+  displayName: Enable Kafka CNAME Registration
+  description: Enable Kafka DNS CNAME Registration
+  value: "false"
```

Comment:
We'll need to make sure that this is set to `true` in AppInterface saas template before merging.

Reply:
@machi1990 - should I open an MR for that now? I assume there is no harm adding vars that are not yet known to the template.

Reply:
> I assume there is no harm adding vars that are not yet known to the template.

I think it should be fine. I remember we chatted about this a while ago, right @JameelB ?

Reply:
If it will be an issue adding a template parameter before it's available, I think we can rework this default logic temporarily to default `ENABLE_KAFKA_CNAME_REGISTRATION` to the value of `ENABLE_KAFKA_CNAME_REGISTRATION`. Let me know what you think.

Reply:
> Does OpenShift template allow for a default value to be set using a parameter?

I don't think so. It would need to be done in the Go code if necessary.

Reply:
Okay. Let's go ahead with adding the parameter with the value set to `true` in AppInterface first.